tests/logger_tests.cpp

```diff
@@ -84,3 +84,22 @@ TEST(Adaptor, EnviromentPath) {
   auto p = log_mr.allocate(100);
   log_mr.deallocate(p, 100);
 }
+
+TEST(Adaptor, STDOUT) {
+  rmm::mr::cuda_memory_resource upstream;
```

Comment:
Tests need to EXPECT or ASSERT something, otherwise they can't fail.

Reply:
Done.

Reply:
Cool, didn't know about this gtest feature to capture stdout/stderr!